fourth_task/src/Client.ts

```diff
@@ -20,7 +20,12 @@ export class Client {
   }
 
   onShip(shipment: ShipmentInterface): string {
-    const { shipmentId, fromAdress, fromZipCode, toZipCode, toAddress } = shipment;
-    return `${shipmentId}, from ${fromAdress} ${fromZipCode}, to ${toAddress} ${toZipCode}, Cost ${this.strategyManager.getCost()}`;
+    const { shipmentId, fromAdress, fromZipCode, toZipCode, toAddress, marksMessages  } = shipment;
+    const basicMessage = `${shipmentId},
+      from ${fromAdress} ${fromZipCode},
+      to ${toAddress} ${toZipCode},
+      Cost ${this.strategyManager.getCost()}`;
+
+    return marksMessages ? basicMessage + marksMessages : basicMessage;
```

Comment:
@dianalemen, this method should know nothing about `markMessages`

Reply:
@AlexHladin what is another way to check if we have marks or have not?

Reply:
@dianalemen, you don't need to check if have marks or not, set the default value [] in your decorator. If some mark value exists in marks decorator will print it.

Reply:
@AlexHladin so, we don't need to add this in onShip method, the decorator for marks messages is going to work separately, yes?